glean-core/ios/Glean/Glean.swift

```diff
@@ -549,6 +549,15 @@ public class Glean {
         }
     }
 
+    /// Set the source tags to be applied as headers when uploading pings.
+    ///
+    /// If any of the tags is invalid nothing will be set and this function will
+    /// return `false`, although if we are not initialized yet, there won't be any validation.
+    ///
+    /// This is only meant to be used internally by the `GleanDebugActivity`.
+    ///
+    /// @param tags A list of tags, which must be valid HTTP header values.
+    ///
```

Comment:
```suggestion
    /// - parameters:
    ///    * tags: A list of tags, which must be valid HTTP header values.
```

Yes, Swift is unusual here.

Reply:
Oops, 🍝